crates/valence_nbt/src/compound.rs

```diff
@@ -137,21 +137,71 @@ where
         self.map.insert(k.into(), v.into())
     }
 
+    #[inline]
```

Comment:
Let's not add `#[inline]` for consistency with the other methods. (I believe a recent Rust version made this unnecessary anyway)